### ISSUE TITLE
Combine bottom bar and top bar

### ISSUE THREAD

**Issue by littlehelper** [nrenner/brouter-web#117]
Hi,
brouter.damsy.net provides a good interface on mobile devices, thanks.
To further optimize/maximize the map view I would suggest to somehow integrate the content of the bottom bar into the top bar by combining them and to remove the bottom bar. At least on (smaller) widescreens this would lead to less prominent currently almost empty bars shown on the screen "hiding" the map.
greetings

**Comment by bagage** [comment]
Yes I agree that the top navbar wastes space on mobile; not sure how though.

**Comment by nrenner** [comment]
In landscape orientation the map indeed looks a bit too narrowed down by the header and footer.

But with the additional stats for "car-eco" profile the footer space is actually needed, see this screenshot of current master simulating Galaxy S5 (640x360):

![screenshot](https://user-images.githubusercontent.com/1092030/38668318-1cac8fae-3e44-11e8-8493-e0dd65d02976.png)

Not sure either what to do about this, combining header and footer specifically for landscape mode would probably not be straightforward to implement and add complexity.

**Comment by bagage** [comment]
I believe the best solution would be to remove the black opaque headerbar and only keep the drawer button when it is collapsed?
